apps/portals/elportal/src/config/synapseConfigs/cohortbuilder.ts

```diff
@@ -117,6 +117,7 @@ export const individualsViewQueryWrapperPlotNavProps: QueryWrapperPlotNavProps =
 export const filesViewQueryWrapperPlotNavProps: QueryWrapperPlotNavProps = {
   rgbIndex,
   name: 'Data',
+  showAccessColumnHeader: true,
```

Comment:
Could you also add the change to the 2 `QueryWrapperPlotNav` instances on the ELITE Portal's StudyDetailsPage?